data/registry/instrumentation-java-struts.yml

```diff
@@ -10,6 +10,6 @@ description: This package provides an instrumentation library for Struts
 authors:
   - name: OpenTelemetry Authors
 urls:
-  repo: https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/struts-2.3
+  repo: https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/struts/
```

Comment:
@trask - does this look ok to you? (AFAIR you added this entry.)

Reply:
yes this looks good!

Reply:
Thanks for confirming.